js/build_map.js

```diff
@@ -134,32 +163,70 @@ function fillMap() {
   updateCounties('USA');
 }
 
-function addButtons() {
-  var buttonContainer = d3.select('.svg-container')
-    .append('div')
-    .attr('id', 'button-container');
-    
-  buttonContainer.append('div')
-    .classed('select-arrowbox', true);
+function resize() {
```

Comment:
❤️ this function

Reply:
I'm kind of thinking it needs a standalone file though.

Reply:
good idea. it got pretty big pretty quick.